client/tests/AccountForm.test.js

```diff
@@ -0,0 +1,178 @@
+import {h} from 'preact';
+/** @jsx h */
+import React from 'react';
+import ReactDOM from 'react-dom';
```

Comment:
I think these react imports could be pulled also since not using in the component